.travis.yml

```diff
@@ -27,7 +27,7 @@ stages:
 jobs:
   include:
     - stage: basic tests
-      script: pytest icepyx/ --verbose --cov app --ignore icepyx/tests/test_behind_NSIDC_API_login.py
+      script: pytest icepyx/ --verbose --cov app --ignore=icepyx/tests/test_behind_NSIDC_API_login.py --ignore=icepyx/tests/test_auth.py
```

Comment:
it would be helpful for future PRs to not include changes that are in another PR (#563)

Reply:
Yes but… It is quite useful when CI is already failing when the PR is created.  PRs that fail are almost never reviewed and merged.